src/helpers.js

```diff
@@ -33,6 +33,10 @@ function getWindowFromNode(node) {
   } else if (node.window) {
     // node is window
     return node.window
+  } else if (node.ownerDocument && !node.ownerDocument.defaultValue) {
```

Comment:
Why are we checking for `defaultValue` here?

Edit: You probably meant `defaultView`. We should really check for an explicit `null` here. I'm not aware of a valid scenario where `defaultView` is `undefined`.

Reply:
Oops, that's my bad sorry. Fixed :)